src/t8_cmesh.h

```diff
@@ -670,13 +670,33 @@ t8_cmesh_get_partition_table (t8_cmesh_t cmesh);
  *                                  the calling processor anymore. Not computed if NULL.
  * \param [out]   first_tree_shared If not NULL, 1 or 0 is stored here depending on whether \a first_local_tree is the
  *                                 same as \a last_local_tree on the next process.
- * \a cmesh must be committed before calling this function. *
+ * \a cmesh must be committed before calling this function. 
  */
 void
 t8_cmesh_uniform_bounds (t8_cmesh_t cmesh, int level, t8_scheme_cxx_t *ts, t8_gloidx_t *first_local_tree,
                          t8_gloidx_t *child_in_tree_begin, t8_gloidx_t *last_local_tree, t8_gloidx_t *child_in_tree_end,
                          int8_t *first_tree_shared);
 
+/**
+ * Calculate the section of a uniform hybrid forest for the current rank
+ * 
+ * \param [in] cmesh        The cmesh to be considered.
+ * \param [in] level        The uniform refinement level to be created.
+ * \param [in] scheme       The element scheme for which to compute the bounds.
+ * \param [out]   first_local_tree  The first tree that contains elements belonging to the calling processor.
+ * \param [out]   child_in_tree_begin The global index of the first element belonging to the calling processor. Not computed if NULL.
+ * \param [out]   last_local_tree  The last tree that contains elements belonging to the calling processor.
+ * \param [out]   child_in_tree_end The global index of the first element that does not belonging to
+ *                                  the calling processor anymore. Not computed if NULL.
+ * \param [out]   first_tree_shared If not NULL, 1 or 0 is stored here depending on whether \a first_local_tree is the
+ *                                 same as \a last_local_tree on the next process.
```

Comment:
I am confused. Need to double check the comment regarding `first_tree_shared`.
Should it be "same as last_local_tree and the **previous** process"?
If so, then needs correction in the original uniform_bounds as well.